lib/logdna/writer.go

```diff
@@ -129,8 +131,12 @@ func extractToken(u *url.URL) (token string, err error) {
 
 func extractTags(u *url.URL, q url.Values) string {
 	tags := q["tag"]
-	list := make([]string, 0, len(tags))
+	if len(tags) == 0 {
+		tags = "";
+	}
+
+	list := make([]string, 0)
 	list = append(list, tags...)
 
-	return strings.Join(list, ",")
+	return fmt.Sprintf("tags=\"%s\"", strings.Join(list, ","))
```

Comment:
`tags` is of type `[]string` right? (`url.Values` is `map[string][]string`).

Could we rewrite the function as
```go
return fmt.Sprintf("tags=\"%s\"", strings.Join(q["tag"], ","))
```
?